### DISCUSSION TITLE
Code view broken

### DISCUSSION THREAD

**Discussion by noritada** [community/community#110293] [Other Feature Feedback, Questions, & Ideas] [answered]
Code views are sometimes broken in recent app versions.
I cannot reproduce this issue intentionally, but I have seen this issue at least three times.

The broken view cannot be fixed with scrolling. Only after switching to another app and switching again to GitHub, I can see the view fixed.

See images to understand the situation.

![image](https://github.com/community/community/assets/20850/b2c49b45-f17b-4aaa-883d-4d0bf1279dcf)

Issue in an older app version on 2024-02-12:
![image](https://github.com/community/community/assets/20850/cd2a48ea-11b9-499a-b49e-71c6c4c589ee)

<details>
<summary>ℹ️ Additional Debug Info</summary>
GitHub for iOS, バージョン 1.150.0 (123387055)
Device: iPhone SE (2nd generation) (iOS 16.7.2)
</details>

**Comment by stevepopovich** [comment, marked as answer]
Hi @noritada we are aware of this issue. Please look out for a fix in the next few days. Thank you for writing in!
